Automattic/wp-calypso
# Command Palette: Command Palette App

Related to:
- https://github.com/Automattic/dotcom-forge/issues/5493
Depends on:
- https://github.com/Automattic/wp-calypso/pull/87790
## Proposed Changes

* Create a command palette app that can be consumed.


## Testing Instructions



* Open the live link and check that there are no regressions when using the command palette.
* On Atomic sites you can load the command palette with https://github.com/Automattic/jetpack/pull/35635
* Check that the command palette works on Atomic sites as expected (the command list isn't complete)
* Load the code from https://github.com/Automattic/jetpack/pull/35635 on your sandbox and check on wp-admin that the command palette is working.

## Pre-merge Checklist
